e2e/ci_tests_test.go

```diff
@@ -151,7 +151,7 @@ var _ = Describe("E2E TEST:"+checks.CheckCITests, func() {
 			}
 			expected := scut.TestReturn{
 				Error:         nil,
-				Score:         2,
+				Score:         3,
```

Comment:
Do we know why this value changed?

Reply:
It was always that way, but it was merged because the test doesn't run on merge (token stuff).